app/api/helpers/files.py

```diff
@@ -84,7 +84,6 @@ def create_save_resized_image(image_file, basewidth=None, maintain_aspect=None,
         im = Image.open(image_file)
     except IOError:
         raise IOError("Corrupt/Invalid Image")
-
```

Comment:
1. Unnecessary change. 
2. We need that space in order to group the logic. 
please add it again. Moreover please squash your commits.

Reply:
Squash commits while merging.